torch2trt/converters/getitem.py

```diff
@@ -117,30 +130,210 @@ def convert_tensor_getitem(ctx):
                 i += 1
             elif isinstance(s, int) or isinstance(s, IntWrapper):
                 # remove int dim
-                i += 1  
+                i += 1
             else:
                 # insert None dim
                 final_shape.append(make_int_wrapper(1))
-                
+
         final_shape = make_size_wrapper(final_shape)
 
         layer = ctx.network.add_shuffle(output_trt)
         layer.reshape_dims = tuple(output.shape) # exclude batch
         layer.set_input(1, final_shape._trt)
         output_trt = layer.get_output(0)
-        
+
     output._trt = output_trt
-    
-    
+
+
 class LambdaModule(torch.nn.Module):
     def __init__(self, fn):
         super(LambdaModule, self).__init__()
         self.fn = fn
-        
+
     def forward(self, x):
         return self.fn(x)
-    
-    
+
+
+@add_module_test(torch.float32, torch.device('cuda'), [(3, 2, 4)], max_batch_size=3)
+def test_tensor_getitem_0d_colon():
+    return LambdaModule(lambda x: x[:])
+
+
+@add_module_test(torch.float32, torch.device('cuda'), [(3, 2, 4)], max_batch_size=3)
+def test_tensor_getitem_0d_ellipsis():
+    return LambdaModule(lambda x: x[...])
+
+
+@add_module_test(torch.float32, torch.device('cuda'), [(3, 2, 4)], max_batch_size=3)
+def test_tensor_getitem_0d_int():
+    return LambdaModule(lambda x: x[0])
+
+
+@add_module_test(torch.float32, torch.device('cuda'), [(3, 2, 4)], max_batch_size=3)
+def test_tensor_getitem_0d_int_colon():
+    return LambdaModule(lambda x: x[0, :])
+
+
+@add_module_test(torch.float32, torch.device('cuda'), [(3, 2, 4)], max_batch_size=3)
+def test_tensor_getitem_0d_int_ellipsis():
+    return LambdaModule(lambda x: x[0, ...])
+
+
+@add_module_test(torch.float32, torch.device('cuda'), [(3, 2, 4)], max_batch_size=3)
+def test_tensor_getitem_0d_1tuple():
+    return LambdaModule(lambda x: x[(0,)])
+
+
+@add_module_test(torch.float32, torch.device('cuda'), [(3, 2, 4)], max_batch_size=3)
+def test_tensor_getitem_0d_2tuple():
+    return LambdaModule(lambda x: x[(0, 1)])
+
+
+# There is currently an issue with this test case.
+# Need to investigate this more.
+#  @add_module_test(torch.float32, torch.device('cuda'), [(3, 2, 4)], max_batch_size=3)
+#  def test_tensor_getitem_0d_3tuple():
+    #  return LambdaModule(lambda x: x[(0, 1, 2)])
```

Comment:
@jaybdub Looks like there's an issue with this test case, which we'll need to look into more. I don't _think_ it's related to the changes in this PR, however, but with how we're indexing in general.